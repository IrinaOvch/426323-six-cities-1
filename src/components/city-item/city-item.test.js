import React from 'react';
import renderer from 'react-test-renderer';

import CityItem from './city-item.jsx';

it(`should render CitiesItem correctly`, () => {
  const page = renderer.create(
      <CityItem
        city={`Paris`}
        handleCityClick={jest.fn()}
        isActiveCity={true}
      />
  ).toJSON();

  expect(page).toMatchSnapshot();
});
