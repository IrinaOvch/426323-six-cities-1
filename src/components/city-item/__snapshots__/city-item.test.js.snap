// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CitiesItem correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
